test/models/podcast_import_test.rb

```diff
@@ -226,4 +231,8 @@ def stub_requests
     with(body: /prxUri/,
          headers: { 'Authorization' => 'Bearer thisisnotatoken' }).
     to_return(status: 200, body: json_file('transistor_episode'), headers: {})
+
+stub_request(:get, "https://feeder.prx.org/api/v1/authorization/episodes/153e6ea8-6485-4d53-9c22-bd996d0b3b03").
```

Comment:
Why is this added now?

Reply:
none of the existing tests check the Feeder episode created for a story, so I had to stub that method to add this one on line 97: `f.distributions.first.get_episode.itunes_type.must_equal 'full'`